cluster/charts/rook-ceph/templates/psp.yaml

```diff
@@ -163,18 +163,5 @@ subjects:
   - kind: ServiceAccount
     name: rook-csi-rbd-provisioner-sa
     namespace: {{ .Release.Namespace }} # namespace:operator
----
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: rbd-csi-nodeplugin
-roleRef:
-  kind: ClusterRole
-  name: rbd-csi-nodeplugin
-  apiGroup: rbac.authorization.k8s.io
-subjects:
-  - kind: ServiceAccount
-    name: rook-csi-rbd-plugin-sa
-    namespace: {{ .Release.Namespace }} # namespace:operator
```

Comment:
This particular ClusterRoleBinding is not related to pod security policies but was included here accidentally. For CSV, which omits PSP, this was a problem, so it had to be moved.

Reply:
Now taken care of by https://github.com/rook/rook/pull/9201